testutil/keeper/proof.go

```diff
@@ -99,7 +100,7 @@ func ProofKeeper(t testing.TB) (keeper.Keeper, context.Context) {
 // directly by the proof keeper or its dependencies.
 //
 // TODO_CONSIDERATION: can we remove the bank keeper as a dependency of the proof keeper?
```

Comment:
For these comments I think by default the account and bank keeper are automatically added, if not used remove them - if it breaks add them back but I am 99.99% sure this is just an ignite thing.